.changeset/empty-coins-lick.md

```diff
@@ -0,0 +1,12 @@
+---
+'@commercetools-uikit/secondary-icon-button': patch
+'@commercetools-uikit/secondary-button': patch
+'@commercetools-uikit/primary-button': patch
+'@commercetools-uikit/rich-text-utils': patch
+'@commercetools-uikit/checkbox-input': patch
+'@commercetools-uikit/icon-button': patch
+'@commercetools-uikit/icons': patch
+'@commercetools-uikit/design-system': patch
+---
+
+New size and size-names introduced for icons, buttons & IconButtons
```

Comment:
It would be great to also include here that we're deprecating some value names and also some design tokens.

Reply:
addressed [here](https://github.com/commercetools/ui-kit/pull/2830/files#diff-6e938d9c08323975ef0e88bd5e6cd151540b1a0344c267a23d37c428bfb28a22R12) @CarlosCortizasCT